item-system/demonlord.js

```diff
@@ -0,0 +1,151 @@
+mergeObject(CONFIG, {
```

Comment:
```suggestion
/*
Support for the Shadow of the Demon Lord system was initially added by Xacus
 */

mergeObject(CONFIG, {
```

Also, add a Credits section to the bottom of the README.md to add credit to yourself for adding this support...
```
# Credits
Support for the Shadow of the Demon Lord system was added by [Xacus](https://github.com/Xacus)
```
(feel free to adjust the wording, link elsewhere, and/or change the credit name to however you'd prefer to be credited)